Neos.ContentRepository.TestSuite/Classes/Behavior/Features/Bootstrap/CRTestSuiteTrait.php

```diff
@@ -133,20 +134,32 @@ protected function readPayloadTable(TableNode $payloadTable): array
      */
     public function iExpectTheContentStreamToNotExist(string $rawContentStreamId): void
```

Comment:
this thing here which was introduced with https://github.com/neos/neos-development-collection/commit/433f3698d0c66499666f9a31fdd60d63e6f38c04 doesnt tell the truth. It should be `assertFalse` in the "old" state but now it actually checks that the contentstream exists.

The underlying problem is in the removed state. in this case we want to check that the contentstream does not exist with `removed FALSE` 

>  when pruning content streams, NO_LONGER_IN_USE content streams will be properly cleaned from the graph projection.

and also the latter part is actually not ensured by just looking at the contentstreams ... i think we/i was a bit uncarefull there.

Reply:
The cr api will now only expose content streams which are not removed. 
See also slack: https://neos-project.slack.com/archives/C04PYL8H3/p1716906008795729?thread_ts=1716905442.715929&cid=C04PYL8H3